server/src/main/java/org/exoplatform/chat/services/mongodb/UserServiceImpl.java

```diff
@@ -493,6 +504,121 @@ private SpaceBean getSpace(String roomId, String dbName)
     return users;
   }
 
+  @Override
+  public List<UserBean> getSuggestionUsers(String currentUserName, String roomId, String filter, String dbName, int limit) {
```

Comment:
Yes, there is one more rule in assigning a task in chat application:

> In Chat add-on, when assigning a task to someone, the autocompletion of the field "Assign to" should list the member of the team first (1) and then my connections (2), then other users (3)

That's why I have to add "roomId" here.